setup.cfg

```diff
@@ -1,33 +0,0 @@
-[bumpversion]
-current_version = 4.11.0dev
-commit = False
-tag = False
-parse = (?P<major>\d+)\.(?P<minor>\d+)\.(?P<patch>\d+)((?P<release>[a-z]+))?((?P<build>\d+))?
-serialize =
-	{major}.{minor}.{patch}{release}{build}
-	{major}.{minor}.{patch}{release}
-	{major}.{minor}.{patch}
-
-[bumpversion:part:release]
-optional_value = stable
-first_value = stable
-values =
-	stable
-	dev
-	a
-	b
-
-[bumpversion:part:build]
-first_value = 0
-
-[bumpversion:file:setup.py]
-search = version = "{current_version}"
-replace = version = "{new_version}"
-
-[bumpversion:file:./galaxy_ng/app/__init__.py]
-search = version = "{current_version}"
-replace = version = "{new_version}"
-
-[bumpversion:file:./galaxy_ng/__init__.py]
-search = __version__ = "{current_version}"
-replace = __version__ = "{new_version}"
```

Comment:
Or maybe this, not sure which one the bump2version uses

Reply:
this one seems to be the one Makefile uses, as the version is right